### PR TITLE
Creating User.cs and Product.cs models

### PR DESCRIPTION
**Pull request by caseywreed**
## Status
**READY**

## Migrations
NO

## Description
Casey and Yonathan created the model files for Users and Products

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()

## Resolves Issue Number
#26 
#27 

## Deploy Notes
None

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

```sh
git pull --prune
git checkout <feature_branch>
bundle; script/server
```

1.

## Impacted Areas in Application
List general components of the application that this PR will affect:

*

### CODE REVIEW

File: Models/Product.cs [hunk 1]
```diff
@@ -12,15 +12,24 @@ public class Product
     public int ProductId {get;set;}
 
     [Required]
-    [DataType(DataType.Date)]
```
Comment (megducharme) [inline Models/Product.cs]: I would include this

File: Models/Product.cs [hunk 2]
```diff
@@ -12,15 +12,24 @@ public class Product
...
     public double Price { get; set; }
+
+    [Required]
+    public bool Sold { get; set; }
+
+    [Required]
+    public int UserId { get; set; }
+
+    [Required]
+    public int ProductTypeId { get; set; }
```
Comment (interlink3d) [inline Models/Product.cs]: Is this where we need to include the FK identifier of: "public ICollection<ProductType> ProductType;"   ??

File: Models/User.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    [Key]
+    public int UserId {get;set;}
+
+    [Required]
+    [StringLength(55)]
+    public string FirstName { get; set; }
+
+    [Required]
+    [StringLength(55)]
+    public string LasttName { get; set; }
```
Comment (interlink3d) [inline Models/User.cs]: extra t in "lastt"

File: Models/User.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
```
Comment (interlink3d) [inline Models/User.cs]: need to include 3 FK to Order, Product, PaymentType based on ERD.

**Comment by megducharme** [comment]
LGTM

**Pull request merged**